var/spack/repos/builtin/packages/molgw/package.py

```diff
@@ -50,7 +51,7 @@ class Molgw(MakefilePackage):
 
     def _get_mkl_ld_flags(self, spec):
         mklroot = str(getenv("MKLROOT"))
-        command = [mklroot + "/bin/intel64/mkl_link_tool", "-libs", "--quiet"]
+        command = [mklroot + "/bin/mkl_link_tool", "-libs", "--quiet"]
```

Comment:
Does this even work with OneAPI?

Reply:
Sure! it does.

Reply:
Ok, I see the variable is set in the package.